Ghidra/Framework/Project/src/main/java/ghidra/framework/main/projectdata/actions/VersionControlCheckOutAction.java

```diff
@@ -193,17 +193,17 @@ public void setMessage(String message) {
 				};
 
 				List<DomainFile> failedCheckouts = new ArrayList<>();
-				int progress = 0;
```

Comment:
why not just initialize progress=1?  that and typo, this PR should be a simple 2 character change

Reply:
Because it would stop before the last item.